components/host-dashboard/reports-section/TotalMoneyManagedSection.js

```diff
@@ -0,0 +1,135 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Dollar } from '@styled-icons/boxicons-regular/Dollar';
+import { InfoCircle } from '@styled-icons/boxicons-regular/InfoCircle';
+import { FormattedMessage } from 'react-intl';
+import styled from 'styled-components';
+
+import { formatCurrency } from '../../../lib/currency-utils';
+
+import Container from '../../Container';
+import { Flex } from '../../Grid';
+import StyledCard from '../../StyledCard';
+import StyledTooltip from '../../StyledTooltip';
+import { P, Span } from '../../Text';
+
+const FundAmounts = styled.div`
+  width: 100%;
+  height: 48px;
+  border-radius: 10px;
+  background-color: #9a7bf1;
+  border-right: 350px solid #46347f;
+  padding-top: 10px;
+  padding-left: 5px;
+
+  @media (max-width: 1025px) {
+    height: 130px;
+    background-color: #9a7bf1;
+    border-right: 0px;
+    border-bottom: 40px solid #46347f;
+  }
+`;
+
+const TotalFundsLabel = styled(Container)`
+  display: table-cell;
+  padding-left: 10px;
+  height: 26px;
+  border-radius: 5px;
+  background-color: white;
+  opacity: 80%;
+  vertical-align: middle;
+`;
+
+const TotalMoneyManagedSection = ({ currency, hostMetrics }) => {
+  const {
+    totalMoneyManaged,
+    pendingPlatformFees,
+    pendingPlatformTips,
+    pendingHostFeeShare,
+    platformTips,
+    hostFees,
+    platformFees,
+  } = hostMetrics;
+  const currentAmount = totalMoneyManaged.value * 100;
+  const projectedAmount =
+    (totalMoneyManaged.value + pendingPlatformFees.value + pendingPlatformTips.value + pendingHostFeeShare.value) * 100;
+  const totalCollectiveFunds =
+    (totalMoneyManaged.value - platformTips.value - platformFees.value - hostFees.value) * 100;
+  const totalHostFunds = hostFees.value * 100;
+  return (
+    <StyledCard borderColor="#46347F">
+      <Container pl={27} pr={24} pt={16} pb={16} backgroundColor="#F6F5FF">
+        <P minHeight="16px" fontSize="12px" fontWeight="500" lineHeight="16px" textTransform="uppercase">
+          <Dollar size={14} color="#1300AB" />
+          <FormattedMessage id="TotalMoneyManagedSection.heading" defaultMessage="Total Money Managed" />
```

Comment:
We already have a string for this one

```suggestion
          <FormattedMessage id="Host.Metrics.TotalMoneyManages" defaultMessage="Total Money Managed" />
```